parsec/mca/device/cuda/device_cuda_module.c

```diff
@@ -2552,6 +2559,8 @@ parsec_cuda_kernel_scheduler( parsec_execution_stream_t *es,
     gpu_device = (parsec_device_gpu_module_t*)parsec_mca_device_get(which_gpu);
     cuda_device = (parsec_device_cuda_module_t *)gpu_device;
 
+    parsec_atomic_fetch_add_int64(&gpu_device->super.device_load, gpu_task->load);
```

Comment:
We never update the load on CPU devices, which means that we never load balance. Looks like the breaking change predates this PR, but it is unclear how we are supposed to achieve load balance between CPU and GPUs now.

Reply:
Our current thinking is that this does not cover  an important use-case anymore, so we are willing to simplify the code by dropping this accounting.

Reply:
See change in device.c:132 that disables get_best_device picking CPU chores when a GPU chore is available.